bedrock/urls.py

```diff
@@ -30,7 +30,7 @@
     path("", include("bedrock.sitemaps.urls")),
     path("careers/", include("bedrock.careers.urls")),
     path("exp/", include("bedrock.exp.urls")),
-    path("external/", include("bedrock.externalpages.urls")),
+    path("external/pocket/", include("bedrock.externalpages.pocket_urls")),
```

Comment:
This goes with moving the pocket URLs to being their own module, rather than in `externalpages.urls`